README.md

```diff
@@ -152,10 +152,10 @@ return td.decode(decryptedData) === 'test';
 
 ### .save
 
-Save files retrieved by Penumbra. Downloads a .zip if there are multiple files.
+Save files retrieved by Penumbra. Downloads a .zip if there are multiple files. Returns an AbortController that can be used to cancel an in-progress save stream.
 
 ```ts
-penumbra.save(data: PenumbraFile[], fileName?: string): Promise<void>
+penumbra.save(data: PenumbraFile[], fileName?: string): AbortController
```

Comment:
What's the difference? Just granularity of control?

Reply:
Yeah, that's the sole difference 👍

Reply:
Oh, and it won't zip if it's one file?

Reply:
Yeah, it only zips if there are 2 or more files